Makd.mak

```diff
@@ -460,13 +460,18 @@ UNITTEST_FILES += $(call find_files,.d,,$C/$(SRC),$(TEST_FILTER_OUT))
 $O/fastunittests.d: $(filter-out %_slowtest.d,$(UNITTEST_FILES))
 $O/allunittests.d: $(UNITTEST_FILES)
 
-# 1. find any module named UnitTestRunner
-# 2. get the relative file path from `src` folder
-# 3. replace slashes with dots to get qualified module name
+
+# Default ocean test runner path
+OCEAN_TEST_RUNNER_MODULE=submodules/ocean/src/ocean/core/UnitTestRunner.d
+
+# Test runner module. Uses ocean if available, else the default druntime runner
+# will be used
 TEST_RUNNER_MODULE=$(shell \
-	  find ./ -path */src/*/core/UnitTestRunner.d \
-	| sed -n 's|^.\+/src/\(.\+/UnitTestRunner\).d$$|\1|p' \
-	| tr / .)
+	if test -f "${OCEAN_TEST_RUNNER_MODULE}"; then \
+		echo ${OCEAN_TEST_RUNNER_MODULE}; \
+	else \
+		echo "object"; \
```

Comment:
Also, this should be empty if not found, if you check a bit further down, the `main()` is created only if the module is empty.